### DISCUSSION TITLE
Add ability to recolor number inside stat indicator boxes

### DISCUSSION THREAD

**Discussion by SirLehto** [runelite/runelite#15060] [Ideas]
Something really needed for the Stat Boosts Indicators:

~1. Make it possible to have the "Next Change Counter" set to ALWAYS ON~
~2. Make it possible to have the "Next Change Counter" always be the top indicator box, so every stat boosted follows in a natural line under it (when set to vertical)~
3. Make it possible recolor the number inside the indicator boxes

**Comment by deathbeam** [comment]
Duplicates

**Comment by SirLehto** [comment]
Searching for this in the open issues shows that only point 3 as being something already suggested. I cannot find other open issues relating to the points 1 and 2 - which are the most important ones.

**Comment by deathbeam** [comment]
Hmm cant find them as well, but I am pretty sure I saw 1 before. I guess you can reopen then and remove 3.

**Comment by SirLehto** [comment]
I edited point 3 to only have the recoloring of the number inside the boxes, as I cannot find this either.
Will you be able to open the issue again and label it accordingly? Thanks man.
